_data/releases.yml

```diff
@@ -0,0 +1,18 @@
+#Release versions
+
+legacy:
+  version: "0.0.0"
+  date: "YYYY-MM-DD"
+stable:
+  version: "0.1.0"
+  date: "2020-06-22"
+nightly:
+  version: "0.2-STABLE"
```

Comment:
Why `STABLE` instead of `SNAPSHOT`?  I would expect this to be `0.2.0-SNAPSHOT`.  A nightly build is _not_ something we want to pitch as stable. ;-)